.github/workflows/update-js-and-babel-on-main.yml

```diff
@@ -42,8 +42,10 @@ jobs:
 
     # This is necessary for the TypeScript to contain translations
     # from Gettex.
-    - name: Compiling Babel translations
+    - name: Update and compile Babel translations
       run: |
+        pybabel extract -F babel.cfg -o messages.pot . --no-location --sort-output
```

Comment:
These are now removed from main due to issues, must be removed here too (they did not automatically because we force-pushed main)